esx_service/utils/error_code.py

```diff
@@ -90,11 +93,14 @@ class ErrorCode:
     ErrorCode.PRIVILEGE_ALREADY_EXIST : "Privilege for ({0}, {1}) already exists",
     ErrorCode.PRIVILEGE_INVALID_VOLUME_SIZE : "Volume max size {0}MB exceeds the total size {1}MB",
     ErrorCode.PRIVILEGE_INVALID_ALLOW_CREATE_VALUE : "Invalid value {0} for allow-create option",
+    ErrorCode.PRIVILEGE_REMOVE_NOT_ALLOWED : "Remove privilege for {} {} is not allowed ({})",
 
-    ErrorCode.DEFAULT_DS_NOT_SET : "Default datastore is not set",
+    ErrorCode.DEFAULT_DS_NOT_SET : "Default datastore is not set for vmgroup {}",
     ErrorCode.DS_NOT_EXIST : "Datastore {0} does not exist",
     ErrorCode.INIT_NEEDED: "Please init configuration with 'vmdkops_admin.py config init' before changing it.",
     ErrorCode.SQLITE3_ERROR: "Sqlite3 error - see log for more info",
+    ErrorCode.DEFAULT_DS_NAME_INVALID : "Default datastore name {} is invalid",
+    ErrorCode.DEFAULT_DS_SET_FAILED : "Set default datastore for vmgroup {} failed ({})",
```

Comment:
Should this block of error codes be modified to have a uniform prefix?  The block above has PRIVILEGE_ and the one below has VMODL_ prefix to clearly identify the category of error code.